src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -458,6 +460,57 @@ GUI_TEST_CLASS_DEFINITION(test_7183) {
     // Expected state: UGENE is not crash
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7193_1) {
+    GTUtilsPcr::clearPcrDir(os);
+    //1. Open "samples/FASTA/human_T1.fa".
+    GTFileDialog::openFile(os, dataDir + "samples/FASTA/human_T1.fa");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    //2. Open the PCR OP.
+    GTWidget::click(os, GTWidget::findWidget(os, "OP_IN_SILICO_PCR"));
+
+    //3. Enter the primers: "GGAAAAAATGCTAAGGGC" and "CTGGGTTGAAAATTCTTT".
+    GTUtilsPcr::setPrimer(os, U2Strand::Direct, "GGAAAAAATGCTAAGGGC");
+    GTUtilsPcr::setPrimer(os, U2Strand::Complementary, "CTGGGTTGAAAATTCTTT");
+    //4. Set both mismatches to 9
+    GTUtilsPcr::setMismatches(os, U2Strand::Direct, 9);
+    GTUtilsPcr::setMismatches(os, U2Strand::Complementary, 9);
+    //5. Set 3' perfect match to 3
+    QSpinBox *perfectSpinBox = GTWidget::findSpinBox(os, "perfectSpinBox");
+    GTSpinBox::setValue(os, perfectSpinBox, 3, GTGlobals::UseKeyBoard);
+
+    //6. Click the find button.
+    GTWidget::click(os, GTWidget::findWidget(os, "findProductButton"));
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    //Expected: one result found
+    CHECK_SET_ERR(GTUtilsPcr::productsCount(os) == 22, QString("Expected 19 result instead of %1").arg(QString::number(GTUtilsPcr::productsCount(os))));
+}
+
+GUI_TEST_CLASS_DEFINITION(test_7193_2) {
+    GTFileDialog::openFile(os, dataDir + "samples/FASTA/human_T1.fa");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    //2. Open the PCR OP.
+    GTWidget::click(os, GTWidget::findWidget(os, "OP_IN_SILICO_PCR"));
+
+    //3. Enter the primers: "AAA" and "CCC".
+    GTUtilsPcr::setPrimer(os, U2Strand::Direct, "AAA");
+    GTUtilsPcr::setPrimer(os, U2Strand::Complementary, "CCC");
+
+    //Expected state: there is a warning about forward primer length
+    QLabel *warningLabel = qobject_cast<QLabel *>(GTWidget::findWidget(os, "warningLabel"));
+    CHECK_SET_ERR(warningLabel != nullptr, "Cannot find warningLabel");
+    CHECK_SET_ERR(warningLabel->text().contains("The forward primer length should be between"), "Incorrect warning message");
+
+    const GTLogTracer lt("One of the given do not fits acceptable length. Task cancelled.");
```

Comment:
nit: remove const. We do not use them for local vars (example: in your code you do not use it for the QLabel above)